utils/download_caffe2_models.sh

```diff
@@ -26,11 +26,12 @@ vgg19
 zfnet512
 bvlc_alexnet
 en2gr
+quant_resnet50
 EOF
 )
 
 for model in $MODELS; do
-  for file in predict_net.pbtxt predict_net.pb init_net.pb; do
+  for file in predict_net.pbtxt predict_net.pb init_net.pb init_net.pbtxt; do
```

Comment:
not all models have `init_net.pbtxt` -- won't this cause cause `wget` to fail? And why do we need `init_net.pbtxt` instead of just using `init_net.pb`?